crates/daemon/src/wallet.rs

```diff
@@ -487,20 +540,46 @@ fn seed_and_rpc_on_same_network(rpc: &electrum_client::Client, network: Network)
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::seed;
+    use bdk::database::MemoryDatabase;
     use bdk_ext::keypair;
     use bdk_ext::new_test_wallet;
     use itertools::Itertools;
+    use rand::distributions::Alphanumeric;
     use rand::thread_rng;
+    use rand::Rng;
     use std::collections::HashSet;
+    use std::env;
+    use std::path::Path;
     use tokio_extras::Tasks;
 
-    impl Actor<(), bdk::database::MemoryDatabase> {
-        pub fn new_offline(
+    const IMPORT_SEED: [u8; 256] = [
```

Comment:
🙃 Since it's only used in one test I would just inline it.

Reply:
❓ Why can't it be a random seed with the right length?

Reply:
@luckysori I am asserting on the address. To ensure that from the same import seed we will always derive the same address. Before, it was also used to check the balance (it owns 0.001 btc)

Reply:
Do we need to assert on the address? Maybe it's enough to just check that it changes as that implies that a new wallet has been loaded.

Reply:
what benefits would we get by that - i feel like the test would be less effective without that assertion.

Reply:
I suppose it is valuable to check that importing a wallet is deterministic.

I would then suggest to move the expected seed into the specific test (I believe it is only used once). It's a bit noisy otherwise.